modules/datastore/src/Drush.php

```diff
@@ -186,13 +182,23 @@ public function list($options = [
    * Private.
    */
   private function createRow($uuid, $item) {
+    // Using deprecation helper.
     return [
       'uuid' => $uuid,
       'fileName' => $item->fileName,
       'fileFetcherStatus' => $item->fileFetcherStatus,
-      'fileFetcherBytes' => \format_size($item->fileFetcherBytes) . " ($item->fileFetcherPercentDone%)",
+      'fileFetcherBytes' => DeprecationHelper::backwardsCompatibleCall(
```

Comment:
A little on the fence about whether to use this Deprecation method, just jump to ByteSizeMarkup and break Drupal 10.1 (which is EOL), or skip the change completely for now.